docs/topics/upgrade.md

```diff
@@ -100,3 +100,24 @@ There are known limitations with VMSS cluster-autoscaler scenarios and upgrade.
 ### Cluster-autoscaler + Availability Set
 
 We don't recommend using `aks-engine upgrade` on clusters that have Availability Set (non-VMSS) agent pools `cluster-autoscaler` at this time.
+
+## Forcing an upgrade
+
+The upgrade operation takes an optional `--force` argument :
+
+```shellscript
+-f, --force   
+force upgrading the cluster to desired version. Allows same version upgrades and downgrades.
+```
+
+In some situation, you might want to bypass the aks-engine validation on your api-model versions and cluster nodes versions. This is at your own risks and you should assess the potential harm of using this flag.
+
+The `--force` parameter instructs the upgrade process to :
+
+- ignore normal version validation
+- include __all__ your cluster's nodes (masters and agents) in the upgrade process. 
+- It will __not__ skip nodes that are already on the target version.
```

Comment:
Items in a bulleted list should follow parallel construction. So if one is a complete sentence, they should all be complete sentences, or if one starts with a verb, they all should, and so on. So maybe change the last three to something like:
```
- __not__ to skip nodes that are already on the target version
- allow any Kubernetes versions, ....
- accept downgrade operations
```

(Sorry to be a pedant. I just read [_Dreyer's English_](https://www.goodreads.com/book/show/40063024-dreyer-s-english) and I'm all fired up.)

Reply:
product placement!

Reply:
Ha, I actually thought exactly that, but felt it was better expressed with a full sentence.
Fixing it anyway. My sense for proper english isn't good enough to dispute Dreyer's English :)